prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py

```diff
@@ -0,0 +1,102 @@
+                                    ]
+                                ):
+                                    # We found the statement, no need to continue
+                                    scp_check_deny_all_regions = True
+                                    break
+                                else:
+                                    # We found a statement restricting some regions, we continue to check if there is another one more restrictive
+                                    scp_check_deny_some_regions = True
+
+                            # Allow if Condition = {"StringEquals": {"aws:RequestedRegion": [region1, region2]}}
+                            if (
+                                statement["Effect"] == "Allow"
+                                and "Condition" in statement
+                                and "StringEquals" in statement["Condition"]
+                                and "aws:RequestedRegion"
+                                in statement["Condition"]["StringEquals"]
+                            ):
+                                if (
+                                    organizations_enabled_regions
+                                    == statement["Condition"]["StringEquals"][
+                                        "aws:RequestedRegion"
+                                    ]
+                                ):
+                                    # We found the statement, no need to continue
+                                    scp_check_deny_all_regions = True
+                                    break
+                                else:
+                                    # We found a statement restricting some regions, we continue to check if there is another one more restrictive
+                                    scp_check_deny_some_regions = True
+
+                        if scp_check_deny_all_regions:
+                            report.status = "PASS"
+                            report.status_extended = (
+                                f"SCP policy restricting regions found: {policy.id}"
+                            )
+                            findings.append(report)
+                            return findings
+
+                    if scp_check_deny_some_regions:
+                        report.status = "FAIL"
+                        report.status_extended = f"SCP policy restricting regions found: {policy.id}, but not the configured ones, check config..."
+                        findings.append(report)
+                        return findings
+
+                    report.status = "FAIL"
+                    report.status_extended = (
+                        f"No SCP restricting by regions found for org: {org.id}"
+                    )
+
+            else:
+                report.status = "FAIL"
+                report.status_extended = (
+                    "AWS Organizations is not in-use for this AWS Account"
+                )
```

Comment:
The same as before, this is is out of context here, so we only check if the organization is filtering regions.

Reply:
I need to give this a thought again.

Reply:
In this case, is right since filtering regions affects the account, so this check is correct.